From 91c07d2a06fd4ec33db8d7737bfd73f61aa645fc Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 24 Oct 2012 11:14:56 +0100
Subject: [PATCH] fish: Fix test-events.sh because of new debug message.

Commit 2e90f4312928f332f8997b52be3fe54f20920242 added a debug message
which isn't filtered out by the set of grep -v's in this test.
Therefore this test would fail when run with LIBGUESTFS_DEBUG=1.
---
 fish/test-events.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fish/test-events.sh b/fish/test-events.sh
index b3684a8..79957ed 100755
--- a/fish/test-events.sh
+++ b/fish/test-events.sh
@@ -22,7 +22,7 @@ set -e
 
 rm -f test.out
 
-./guestfish -a /dev/null <<'EOF' | grep -v get_verbose | grep -v get_trace | grep -v 'library .*0x' > test.out
+./guestfish -a /dev/null <<'EOF' | grep -v get_verbose | grep -v get_trace | grep -v 'library .*0x' | grep -v 'library command' > test.out
 trace true
 
 event ev1 * "echo $EVENT $@"
-- 
1.7.11.4

